app/client/stylesheets/_login.scss

```diff
@@ -1,9 +1,18 @@
 #login {
-  height: 100%;
+  height: calc(100% + 49px);
   width: 100%;
-  background-color: $login-bg-color;
-  background:  $login-bg;
+  background-image: url('../assets/images/hero.jpg');
   background-size: cover;
+  background-position: center center;
+
+  &::before {
+    content: '';
+    background-image: linear-gradient(to bottom right, #02ccba 0%, #3d99be 31%, #9848d5 100%);
+    opacity: 0.8;
+    width: 100%;
+    height: calc(100% + 50px);
+    position: absolute;
+  }
 
   > .container {
```

Comment:
no need for a ">" 
the sass compiler will add it in purely to the fact that it's nested